src/api/status/src/server.js

```diff
@@ -8,6 +8,7 @@ const getGitHubData = require('./js/github-stats');
 const getFeedCount = require('./js/feed-stats');
 const getPostsCount = require('./js/posts-stats');
 const getJobCount = require('./js/queue-stats');
+const { version } = require('../../../../package.json');
```

Comment:
I don't think this is going to work. The legacy back-end has access to this file, but microservices are not really aware of what's outside of their container, so the main `package.json` won't be available.
We need to find a different way to get Telescope's version. Also, I think the front-end would need some changes. If you think it's too much for this PR, you can do it in a follow-up.

Reply:
Can definitely do the frontend portion in a follow up, also forgot the purpose of microservices, I'll figure out how to grab the version another way